### PR TITLE
Code Review: Calculator App - Milestone four

### PR DESCRIPTION
**Pull request by Milypm**
For this project, I implement the logic for doing the calculations on the app:

- Create the 'logic' directory inside 'src'
- Create the calculate and operate components (inside 'logic' directory)
- Write several conditions depending on the requested operation/number
- Import and use Big.js library to perform the operations

Tools:
- JavaScript
- Big.js
- React
- React.Dom
- JSX
- Eslint

### CODE REVIEW

File: src/logic/calculate.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import operate from './operate';
+
+const calculate = ({ calcObj, btnName }) => {
+  let { total, next, operation } = calcObj;
```
Comment (Elbie-em) [inline src/logic/calculate.js:4]: - [x] calcObj and btnName should be parameters as they will be passed to calculate as arguments. This illustration implies that you are passing an object and destructuring the two from that object. Kindly fix this.

File: src/logic/operate.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import Big from 'big.js';
+
+const operate = ({ numberOne, numberTwo, operation }) => {
```
Comment (Elbie-em) [inline src/logic/operate.js]: - [x] numberOne, numberTwo, and operation should be parameters as each will be passed to operate as arguments. This illustration implies that you are passing an object and destructuring the three from that object. Your implementation of operate in calculate.js clearly shows you are not passing an object but arguments. Kindly fix this.

**Review by Elbie-em — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED 🟡  ♻️ 

HI @Milypm, you have done a great job in meeting almost all the requirements for this milestone. However, there is one minor change that needs to be addressed before this can be approved. Check my comments below ⬇️ 

*All the best and good luck* 💪 

> Elbie Moonga (Part-time TSE)

> Github (@Elbie-em)
> Slack (Elbie Moonga)
> Twitter ([@ElbieEm](https://twitter.com/ElbieEm))
> LinkedIn([Elbie Moonga](https://www.linkedin.com/in/elbiemoonga/))

**Review by NtwaliHeritier — APPROVED** [review]
### **STATUS: APPROVED👍️👍️**

Hello @Milypm 
Great work making the requested changes.👏️👏️
You can merge👍️👍️

Happy coding💻️💻️

**Pull request merged**